src/components/container/Container.js

```diff
@@ -0,0 +1,31 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Box from '../box';
+import cx from 'classnames';
+import theme from './theme.css';
+
+class Container extends PureComponent {
```

Comment:
PP: don't use PureComponent, it's premature optimisation

Reply:
All the components are PureComponents

Reply:
Yeah they shouldn't be